install/linux/usr/share/odemis/hwtest/technolution-asm.odm.yaml

```diff
@@ -0,0 +1,37 @@
+MultiBeamSEM: {
+    class: Microscope,
+    role: mbsem,
+    children: ["ASM", "EBeamScanner", "MirrorDescanner", "MPPC",],
+}
+
+"ASM": {
+  class: technolution.AcquisitionServer,
+  role: "main",
```

Comment:
If I understand correctly, this component is also useful for the client, right? (eg, to change the external storage). So it should indeed have a role. However, "main" doesn't sound good. What's the role that will be used for the fastem? Maybe "asm" would be fine?

Reply:
Indeed overlook this one. "asm" is good.

Reply:
Ok great, @pieleric I think it can be merged then.